handler.c

```diff
@@ -147,6 +147,14 @@ static const uint8_t EFI_IMAGE_SECURITY_DATABASE2[] = {'d',0,'b',0,'t',0};
 
 #define AUTH_PATH_PREFIX "/var/lib/varstored"
 
+#ifdef AUTH_ONLY_PK_REQUIRED
+#define AUTH_DB_REQUIRED false
```

Comment:
The explanation from the commit message, maybe a short version of it would be good as a comment here?

Reply:
Done